### DISCUSSION TITLE
Ferramenta de Test de integração

### DISCUSSION THREAD

**Discussion by jardianoalmeida** [Flutterando/forum#84] [Dicas]
Alguém conhece uma ferramenta pra teste de integração no Flutter?

**Comment by mateusfccp** [comment]
https://docs.flutter.dev/cookbook/testing/integration/introduction

https://github.com/flutter/flutter/tree/main/packages/integration_test

**Reply by jardianoalmeida** [reply]
procuro uma ferramenta como o https://appium.io/ por exemplo, conhece alguma ?

**Comment by eduardoflorence** [comment]
Você pode usar o próprio Appium com o Flutter, pois já há drive para ele. Veja em: https://github.com/appium-userland/appium-flutter-driver

Aqui tem um artigo que mostra como utilizar: https://blog.flutterando.com.br/flutter-testes-de-integra%C3%A7%C3%A3o-com-appium-41c1a8741f08

**Reply by jardianoalmeida** [reply]
Obrigado pela resposta, eu já usei esse cara, tá descontinuado e foi substuido pelo integration_package

**Reply by jardianoalmeida** [reply]
Mas eu não consegui fazer não, tentei em Java, Python, JS
